src/global-firestore-data/semesters.ts

```diff
@@ -37,61 +20,75 @@ const editSemesters = (
 };
 
 /**
- * Toggles whether semesters are ordered by newest/oldest
- * @returns true iff semesters were previously ordered oldest -> newest,
- *          false otherwise
+ * Sets whether semesters are ordered by newest/oldest
  */
-export const toggleOrderByNewest = (): boolean => {
-  const toggled = !store.state.orderByNewest;
-  store.commit('setOrderByNewest', toggled);
+export const setOrderByNewest = (orderByNewest: boolean): void => {
```

Comment:
Changed `toggle` (which set the negated boolean value) to `set` (which sets a specific boolean value) to simplify the code in the frontend component and unify the `global-firestore-data` functions (i.e. `setBlah(newValue)` or `editBlah(updater)`). It should have the same functionality.

Lmk if you have a different opinion and I can change this back!